update-motd.d/30-pengwin-setup

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+printf " Pengwin include 'Pengwin Setup', allows new Pengwin users to get started on Linux immediately"
```

Comment:
Pengwin includes 'Pengwin Setup,' which allows new Pengwin users to get started on Linux immediately

Reply:
Done